### ISSUE TITLE
Questions about flow data input and output

### ISSUE THREAD

**Issue by packer-c** [tsinghua-fib-lab/KSTDiff-Urban-flow-generation#1]
Thanks for your great work and sharing your code!

I would like to know how you split the dataset. I mean that, In **TABLE 4**, during the pre-training and training, is the input (training set) all one day (24 hours)? And how many days is the test set? I guess using one day's flow data to generate next day, right?

Looking forward to your reply. Thanks a lot!

**Comment by zhou-zl18** [comment]
We split the training set and test set by regions instead of by days.   

For example, as shown in Table 3, in Beijing dataset, we choose 648 regions as train regions and 362 regions as test regions. The training data is the flow in the 648 train regions (which contains 31 days, 2018.01.01-2018.01.31), and the test data is the flow in the 362 test regions (also 31 days).  

You can refer to **Appendix A** for details of train and test regions.

**Comment by packer-c** [comment]
Got it! Thanks for your reply.

By the way, in Table 4, did you use one day's data to generate one day's data? Similarly, in Figure 6, one day's data is used to generate one week, two weeks, and four weeks, right?

**Comment by zhou-zl18** [comment]
In Table 4, we use one day's data to generate one day's data. 
In Figure 6, we use one week's data to train the model, and generate one week, two weeks, and four weeks.

**Comment by packer-c** [comment]
Thanks!
